spec.md

```diff
@@ -1492,6 +1502,187 @@ Applying integer decoding with a divisor of `100` to create an array of 32-bit f
 [ 1.00, 1.00, 1.00, 1.00, 0.50, 0.50 ]
 ```
 
+### Extra data
+
+The following are fields that are __not__ supplied by the *RCSB*, and are provided for applications to transfer non-molecular data along with the molecular data.
+
+Each field is a [map](#types), and there are 6 possible fields `bondProperties`, `atomProperties`, `groupProperties`, `chainProperties`,
+`modelProperties`, and `extraProperties`.  The first 5 fields are restricted to holding data that is relevant to the field's prefix, while
+any other data can be stored in the `extraProperties` field.  
+
+As mentioned in [Types](#types) in some fields we restrict the key-value types to specific types. The current type format the sepc follows is:
+* <String, Array|Binary>
+  * `atomProperties`
+  * `bondProperties`
+  * `groupProperties`
+  * `chainProperties`
+  * `modelProperties`
+* <String, String|Float|Integer|Map|Array|Binary>
+  * `extraProperties`
+
+For each of the fields above, we do reserve some key-value pair formats to help ensure cross-application-compatibility.  If you feel
+a certain key-value pair should be reserved and made a part of the spec, please let us know!  
+
+We encourage you to apply our encoding techniques to your application data to reduce file sizes!
+
+The restrictions on each of the lengths of the values in these fields are __not__ enforced at decode-time but are reserved to help guide applications.
+
+| Name            | length-restrictions     | 
+|-----------------|-------------------------|
+| bondProperties  | length of numBonds      |
+| atomProperties  | length of numAtoms      |
+| groupProperties | length of numGroups     |
+| chainProperties | length of numChains     |
+| modelProperties | length of numModels     |
+| extraProperties | None                    |
+
+Example
+
+```
```

Comment:
Visual detail: I think you can use ` ```python ` for a nice syntax highlighting here. I assume you removed the json highlighting since comments and ... would have looked bad there...

Reply:
ok I will try that!